### ISSUE TITLE
License help menu items from settings

### ISSUE THREAD

**Issue by fcollonval** [jupyterlab/jupyterlab#10341]
With #10254 in, the package can now defined user customizable menus from the settings.

We could consider bringing it for:

https://github.com/jupyterlab/jupyterlab/blob/a8dff39a1a2be850a80861e05145e763079b8794/packages/help-extension/src/index.tsx#L607

This implies to always add the commands and to set their `isVisible` to be condition of the server capability.

What do you think @bollwyvl ?

**Comment by bollwyvl** [comment]
Just to check, `isVisible` on the command will hide the menu item?

**Comment by bollwyvl** [comment]
Actually, the issue is that it _won't even be registered_.

**Comment by fcollonval** [comment]
Yes the items won't be visible. And indeed this implies registering the commands even when they are not available.

**Comment by krassowski** [comment]
Still relevant, but https://github.com/jupyterlab/jupyterlab/pull/17361 changed the location of the code:

https://github.com/jupyterlab/jupyterlab/blob/e1317a31ad5ff88473d533a417ac3258489e3276/packages/apputils-extension/src/licensesplugin.ts#L236-L239
